doc/user/zebra.rst

```diff
@@ -685,6 +685,40 @@ replaces the information sent in the first message.
 If the connection to the FPM goes down for some reason, zebra sends
 the FPM a complete copy of the forwarding table(s) when it reconnects.
 
+.. _zebra-dplane:
+
+Dataplane Commands
+========================
```

Comment:
While you're in here please cut the underline down to the length of the title

Reply:
thanks, pushed a fix